airbrakes/state.py

```diff
@@ -173,8 +174,8 @@ def update(self):
 
         data = self.context.data_processor
 
-        # If our altitude is 0, we have landed:
-        if data.current_altitude <= GROUND_ALTITUDE:
+        # If our altitude and speed are around 0, we have landed
+        if data.current_altitude <= GROUND_ALTITUDE and abs(data.vertical_velocity) <= LANDED_SPEED:
```

Comment:
maybe as a fallback we can have time in free fall state should not be greater than 80 seconds?

Reply:
I'm okay with that